src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateForm.tsx

```diff
@@ -0,0 +1,213 @@
+import { MouseEvent, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import produce from 'immer';
+
+import { quickCreateVM } from '@catalog/utils/quick-create-vm';
+import { isRHELTemplate } from '@catalog/utils/utils';
+import { useWizardVMContext } from '@catalog/utils/WizardVMContext';
+import {
+  ProcessedTemplatesModel,
+  SecretModel,
+  V1Template,
+} from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { updateCloudInitRHELSubscription } from '@kubevirt-utils/components/CloudinitModal/utils/cloudinit-utils';
+import {
+  addSecretToVM,
+  applyCloudDriveCloudInitVolume,
+} from '@kubevirt-utils/components/SSHSecretSection/utils/utils';
+import { RHELAutomaticSubscriptionData } from '@kubevirt-utils/hooks/useRHELAutomaticSubscription/utils/types';
+import { getAnnotation, getResourceUrl } from '@kubevirt-utils/resources/shared';
+import {
+  ANNOTATIONS,
+  getTemplateOS,
+  getTemplateVirtualMachineObject,
+  LABEL_USED_TEMPLATE_NAME,
+  LABEL_USED_TEMPLATE_NAMESPACE,
+} from '@kubevirt-utils/resources/template';
+import { getMemoryCPU } from '@kubevirt-utils/resources/vm';
+import { ensurePath, isEmpty } from '@kubevirt-utils/utils/utils';
+import { k8sCreate } from '@openshift-console/dynamic-plugin-sdk';
+import { useAccessReview, useK8sModels } from '@openshift-console/dynamic-plugin-sdk';
+
+import { uploadFiles } from '../utils';
+
+import useCreateVMName from './useCreateVMName';
+import { useDrawerContext } from './useDrawerContext';
+
+const useCreateForm = (
```

Comment:
This hook name is very generic, but it's for a specific form. Could you make the name more specific?